downstream/modules/automation-hub/proc-create-groups.adoc

```diff
@@ -8,8 +8,7 @@ You can create and assign permissions to a group in Automation Hub that enables
 
 .Prerequisites
 
-* You have *groups* permissions and can create and manage group configuration and access in Automation Hub.  See xref:ref-permissions[Automation Hub permissions] for information on system permissions.
-
+* You have *groups* permissions and can create and manage group configuration and access in Automation Hub.
 .Procedure
```

Comment:
```suggestion

.Procedure
```

Reply:
I think you need a space before .procedure to get the subheader right. Check the preview build to verify